multiply.py

```diff
@@ -0,0 +1,11 @@
+"""
+This is an example docstring.
+
+A docstring with a summary line, a blank line
+and some more text like here.
+"""
```

Comment:
Make sure you update your docstrings. This will satisfy pydocstyle, but it's not terribly useful.

Reply:
Same for mod.py and divide.py

Reply:
These docstrings have been corrected in the functions branch.